src/zherebko/index.js

```diff
@@ -24,12 +24,25 @@ export default function() {
     let maxIndex = 0;
     dag.eachLinks(({ data }) => {
       minIndex = Math.min(minIndex, data.index);
-      maxIndex = Math.max(maxIndex, data.index);
+      maxIndex = Math.max(maxIndex, data.index > 0 ? data.index : 1);
     });
     let maxLayer = ordered.length - 1;
+
     dag.each((node) => {
-      node.x = (-minIndex / (maxIndex - minIndex)) * width;
-      node.y = (node.layer / maxLayer) * height;
+      if (maxLayer === 0) {
+        // If there's just 1 node
+        // we should just center it
+        node.x = width / 2;
+        node.y = height / 2;
+      } else if (minIndex === maxIndex) {
+        // If there are no links, we want to stack the
+        // nodes vertically
+        node.x = (node.layer / maxLayer) * width;
+        node.y = height / 2;
+      } else {
+        node.x = (-minIndex / (maxIndex - minIndex)) * width;
+        node.y = (node.layer / maxLayer) * height;
+      }
```

Comment:
with both of the other changes, this should stay the same as it's handled outside of this loop.